services/pull/patch.go

```diff
@@ -87,6 +87,14 @@ func TestPatch(pr *issues_model.PullRequest) error {
 		}
 	}
 	pr.MergeBase = strings.TrimSpace(pr.MergeBase)
+	if pr.HeadCommitID, err = gitRepo.GetBranchCommitID("tracking"); err != nil {
```

Comment:
Small nit, it could be `gitRepo.GetRefCommitID(git.BranchPrefix + "tracking")` (as above line 84) for consistency, or make them both use the `gitRepo.GetBranchCommitID`.

Reply:
fixed